src/transformer/events/mod_scorm/sco_launched.php

```diff
@@ -21,10 +21,13 @@
 use src\transformer\utils as utils;
 
 function sco_launched(array $config, \stdClass $event) {
+    global $DB;
+
     $repo = $config['repo'];
     $user = $repo->read_record_by_id('user', $event->userid);
     $course = $repo->read_record_by_id('course', $event->courseid);
-    $scorm = $repo->read_record_by_id('scorm', $event->objectid);
+    $scormscoes = $DB->get_record('scorm_scoes', array('id' => $event->objectid), 'scorm');
```

Comment:
This will fail tests, you should fetch records using the `$repo` variable instead. The tests use a different repo that mocks the database using a JSON file so that the tests run quicker since they don't have to spin up a DB.